src/device-monitoring/config/constants.py

```diff
@@ -25,24 +37,22 @@ class ProductionConfig(Config):
 
 
 class DevelopmentConfig(Config):
-    dotenv_path = os.path.join(os.path.dirname(__file__), '.env')
-    load_dotenv(dotenv_path)
     DEVELOPMENT = True
     DEBUG = True
     MONGO_URI = os.getenv("MONGO_DEV_URI")
     DB_NAME = os.getenv("DB_NAME_DEV")
 
```

Comment:
I feel like we should separate the REDIS configs for Dev environment so that we don't have to manually change the value of `REDIS_SERVER` in the `.env` during deployment or local testing. So, we can have have something like `CACHE_REDIS_HOST = os.getenv('REDIS_SERVER_DEV')`. Production and staging can remain `REDIS_SERVER` with a value of `airqo-redis-server` in the `.env`